### DISCUSSION TITLE
Validations only run for one field instead of all fields

### DISCUSSION THREAD

**Discussion by danieljcafonso** [react-hook-form/react-hook-form#2546] [Q&A]
Hey everyone, I have a scenario with 2 number inputs. Here I need for the minValue input value to be less than the maxValue value. I need for the following scenario to happen:

- Step 1 - *Changes max value to 100* : no errors
- Step 2 - *Changes min value to 101* : Error , as the the min value is more than the max value
- Step 3 - *Changes max value to 102*: The min value error disappears.

The schema I'm providing to my yupResolver is the following:
```js
  yup
    .object()
    .shape({
      maxValue: yup
        .number()
        .transform((value) => (isNaN(value) ? undefined : value))
        .required(maxValueErrorMessage),
      minValue: yup
        .number()
        .transform((value) => (isNaN(value) ? undefined : value))
        .lessThan(yup.ref('maxValue'), minValueErrorMessage)        
        .required(minValueErrorMessage),
    })
    .required();
```

The thing here is that Step 1 and Step 2 seem to work out fine, but, when I perform the step 3 the minValue error doesn't disappear. I'm suspecting that the validations are only running for the changed field instead of running for everything. Is there any way for me to force the validations to run for all the fields?

**Comment by bluebill1049** [comment]
if you want to run the entire form, you can use `trigger()`
